README_shelltest.md

```diff
@@ -0,0 +1,36 @@
+# CLI Testing
```

Comment:
The instructions are somewhat helpful for a developer, but I'm not sure a shelltest readme file is necessary in the final product. Maybe add this to the tests directory, then add an instruction here to run everything from the root directory.